### PR TITLE
Enable multi notary deployment

### PR DESCRIPTION
**Pull request by suvajit-sarkar**
Signed-off-by: suvajit-sarkar <suvajit.sarkar@accenture.com>

**Changelog**
- Add multiple notary option in network.yaml
- Update nmap chart logic to create network-initial-param with multiple notaries
- Fix git SSH protocol not working 

 

**Reviewed by**
@jagpreetsinghsasan 

 

**Linked issue**
#1222

### CODE REVIEW

File: platforms/r3-corda-ent/charts/nmap/templates/deployment.yaml [hunk 1]
```diff
@@ -72,40 +72,39 @@ spec:
...
+
+              fi
               done
+              echo "]" >> $file
+              echo "minimumPlatformVersion = 1" >> $file
+              echo "maxMessageSize = 10485760" >> $file
+              echo "maxTransactionSize = 10485760" >> $file
+              echo "eventHorizonDays = 1" >> $file
+
+              cat $file
```
Comment (jagpreetsinghsasan) [inline platforms/r3-corda-ent/charts/nmap/templates/deployment.yaml:106]: Is this required?

File: platforms/r3-corda-ent/charts/nmap/templates/deployment.yaml [hunk 2]
```diff
@@ -220,7 +219,7 @@ spec:
                 echo "crl certificates might not have been put in vault. Giving up!!!"
                 exit 1
               fi
-              echo "Done"
+              echo "Done All certificates"
```
Comment (jagpreetsinghsasan) [inline platforms/r3-corda-ent/charts/nmap/templates/deployment.yaml:222]: `All` should be `all`

File: platforms/r3-corda-ent/configuration/samples/network-cordaent.yaml [hunk 3]
```diff
@@ -143,28 +143,29 @@ network:
...
-            targetPort: 10005
-          dbtcp:
-            port: 9101
-            targetPort: 1521
-          dbweb:             
-            port: 8080
-            targetPort: 81
+        notaries:
+          - notary:
+            name: notary-1
```
Comment (jagpreetsinghsasan) [inline platforms/r3-corda-ent/configuration/samples/network-cordaent.yaml:148]: Its good to show multiple notaries in sample

**Review by jagpreetsinghsasan — CHANGES_REQUESTED** [review]

File: platforms/r3-corda-ent/charts/nmap/templates/deployment.yaml (hunk 1 above)
Comment (suvajit-sarkar) [reply platforms/r3-corda-ent/charts/nmap/templates/deployment.yaml:106]: cat is not required just wanted to keep it for log checking.. will remove it

**Review by sownak — CHANGES_REQUESTED** [review]

File: platforms/r3-corda-ent/charts/nmap/templates/deployment.yaml [hunk 4]
```diff
@@ -72,40 +72,39 @@ spec:
...
+
+                echo "  {" >> $file
+                echo "    notaryNodeInfoFile: \"notary-nodeinfo/${notary_nodeinfo_name}\"" >> $file
+                echo "    validating = \"${validating}\"" >> $file
+                echo "  }" >> $file
+
+              fi
               done
+              echo "]" >> $file
+              echo "minimumPlatformVersion = 1" >> $file
```
[the remark this answers is not in the document]
Comment (suvajit-sarkar) [reply platforms/r3-corda-ent/charts/nmap/templates/deployment.yaml:101]: yes sure 4 should be there

**Review by sownak — APPROVED** [review]
LGTM

**Comment by sownak** [comment]
@all-contributors please add @suvajit-sarkar for code and documentation

**Pull request merged**